packages/examples/cvat/exchange-oracle/src/crons/cvat/state_trackers.py

```diff
@@ -150,8 +127,17 @@ def track_assignments(logger: logging.Logger) -> None:
 
 
 @cron_job
-def track_completed_escrows(logger: logging.Logger) -> None:
-    handle_completed_escrows(logger)
+def track_completed_escrows(logger: logging.Logger, session: Session) -> None:
+    awaiting_validations = cvat_service.create_escrow_validations(session)
+    logger.info(
+        f"{len(awaiting_validations)} escrow validations "
+        f"are in awaiting state: {awaiting_validations}"
```

Comment:
```suggestion
        f"Got {len(awaiting_validations)} new escrows"
        f"awaiting validation: {format_list(awaiting_validations)}"
```

Also, please change this (and other updated) loggings to full list in the debug mode and to `format_list()`-formatted for the `info` loglevel.